docs/README.md

```diff
@@ -56,6 +56,26 @@ source bcml_env/bin/activate # will activate the venv
 pip install bcml
 ```
 
+**Full Linux Example**
+
+`sudo pacman -S python39` Adjust for you distribution, arch defaults to a newer python
+
+```mkdir -p ~/.local/share/cemu/graphicPacks/BreathOfTheWild_BCML
+python3.9 -m venv /.local/bcml_env
+source ~/.local/bcml_env/bin/activate
+pip install bcmL
+~/.local/bcml_env/bin/bcml
+```
+
+to launch BCML in the future
+
+`source ~/.local/bcml_env/bin/activate; ~/.local/bcml_env/bin/bcml`
+
+- In BCML, check 'without cemu' & 'disable links for master mod'; set export path to '~/.local/share/cemu/graphicPacks/BreathOfTheWild_BCML'
```

Comment:
Why is disabling links necessary for this? Is there an error?

Reply:
It was a suggestion in https://github.com/NiceneNerd/BCML/issues/421#issuecomment-1281246780. I'm not sure if it's actually required, but it worked for me on garuda (arch with extra GUI tools)

Reply:
I use Garuda and don't use the no links option. That should generally be avoidable.

Reply:
Your right, just tried and remerged.  Moved that line to "if it doesn't work" step